pkg/detectors/artifactory/artifactory.go

```diff
@@ -52,6 +54,7 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 		if len(URLmatch) != 2 {
 			continue
 		}
+
 		resURLMatch = strings.TrimSpace(URLmatch[1])
```

Comment:
On a separate note, this detector appears to ignore all other URL matches other than the last one matched. https://github.com/trufflesecurity/trufflehog/blob/b2311b4ad2169f5d6c24e3ad136ce698ecfab042/pkg/detectors/mite/mite.go#L52 is how we should be handling this; nested loop w/ matches and urlMatches. Please correct me if I'm off basis here @mcastorina

Reply:
That's correct - we noticed that oddity when we were updating this detector but decided it was out of scope for this PR.

Reply:
Gotcha, in that case I think this is okay to merge